### PR TITLE
🎉 Google Analytics: support incremental sync

### PR DESCRIPTION
**Pull request by vitaliizazmic**
## What

Setting installing Google Analytics Tap from incremental_mode branch of vitaliizazmic fork for testing new incremental changes.  

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

### CODE REVIEW

**Comment by sherifnada** [comment]
@vitaliizazmic the failure was being caused by a bug in standard tests -- please proceed with publishing the connector

File: airbyte-integrations/connectors/source-googleanalytics-singer/setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
     # two sets of dependencies: 1) for main 2) for standard test deps. 2 does not have all of the dependencies of 1, which is we cannot use install_requires.
     extras_require={
         "main": [
-            "pipelinewise-tap-google-analytics @ https://github.com/airbytehq/pipelinewise-tap-google-analytics/tarball/master#egg=pipelinewise-tap-google-analytics-1.2.2-airbyte",
+            "pipelinewise-tap-google-analytics @ https://github.com/vitaliizazmic/pipelinewise-tap-google-analytics/tarball/incremental_mode#egg=pipelinewise-tap-google-analytics-1.3.0-airbyte",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-googleanalytics-singer/setup.py:37]: this should be pointing to the Airbyte fork before merging

**Review by sherifnada — CHANGES_REQUESTED** [review]
I don't think this was tested correctly -- the configured catalogs have not changed, and the catalog output by the connector has not changed I believe? Shouldn't those have changed?

**Review by vitaliizazmic — COMMENTED** [review]
Tap fork changed

**Review by sherifnada — APPROVED** [review]

**Pull request merged**